woocommerce/woocommerce-gutenberg-products-block
# Remove empty pattern file

## What

This PR removes an empty pattern file.

## Why


It was left empty by mistake and it's causing an error when trying to register the empty pattern.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Checkout to this branch.
2. Make sure you are not getting a PHP error



## Screenshots or screencast




## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog
